File: Java/TP4-4/src/Regular.java
public class Regular extends Usuario {

	public Regular(String login) {
		super(login);
	}

	public void menuRegular() {
		System.out.println("Menu:");
		System.out.println("1-Permissões");
		System.out.println("2-Dashboard");
		System.out.println("8-Trocar de Usuário");
		System.out.println("9-Sair");
		System.out.println("Opção: ");
	}

	@Override
	public void despedidaUsuario() {
		System.out.println("Log-off " + super.getLogin());
	}

	@Override
	public void descricaoUsuario() {
		System.out.println("Pode consultar suas informações.");
	}

	public void dashboard() {
		System.out.println("Usuario: " + super.getLogin());
		System.out.println("Tipo: Regular");
		System.out.print("Permissoes: ");
		this.descricaoUsuario();
	}
}

File: Kotlin/TPK4-4/src/Regular.kt
class Regular(login:String):Usuario(login) {
  fun menuRegular() {
    println("Menu:")
    println("1-Permissões")
    println("2-Dashboard")
    println("8-Trocar de Usuário")
    println("9-Sair")
    println("Opção: ")
  }
  fun despedidaUsuario() {
    System.out.println("Log-off " + super.getLogin())
  }
  fun descricaoUsuario() {
    println("Pode consultar suas informações.")
  }
  fun dashboard() {
    System.out.println("Usuario: " + super.getLogin())
    println("Tipo: Regular")
    print("Permissoes: ")
    this.descricaoUsuario()
  }
}